### PR TITLE
Do not delete zones. Mark them as "not enabled" instead.

### PR DESCRIPTION
**Pull request by ekoval**
Closes: #7 #6 

1. Do not delete zones. Mark them as "not enabled" instead.
2. Add "data_type" and "zone_id" fields to tracking data.

### CODE REVIEW

File: src/models.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+    Document, ObjectIdField, StringField, FloatField, IntField, DateTimeField,
+    BooleanField)
 from datetime import datetime
 
+from constants import DATA_TYPES
+
 
 class Tracking(Document):
     tracking_id = StringField(required=True)
+    data_type = StringField(max_length=5, choices=DATA_TYPES, required=True)
```
Comment (alexsv) [inline src/models.py:11]: i'm sceptic about choices attr here, which accepts list of tuples

File: src/app.py [hunk 2]
```diff
@@ -56,7 +56,9 @@ def post_zone(data):
 @app.route('/v1/zones/<id>', methods=['DELETE'])
 @api_response()
 def delete_zone(id):
-    Zone.objects.get(pk=id).delete()
+    zone = Zone.objects.get(pk=id)
+    zone.enabled = False
```
Comment (alexsv) [inline src/app.py:60]: maybe add a field - deleted ?
i thought about enabled to be able to enable/disable the zone temporarly
deleted means delete it forever

File: src/models.py (hunk 1 above)
Comment (ekoval) [reply src/models.py:11]: Why?

File: src/app.py (hunk 2 above)
Comment (ekoval) [reply src/app.py:60]: Agree

Comment (ekoval) [reply src/app.py:60]: Done

**Review by alexsv — APPROVED** [review]

**Pull request merged**